packages/components/src/components/segmented-control/segmented-control-item/segmented-control-item-styles.ts

```diff
@@ -64,32 +65,49 @@ export const getComponentCss = (
         display: 'block',
         height: '100%',
         width: '100%',
-        padding: `${pxToRemWithUnit(12)} ${ITEM_PADDING}`,
+        padding: hasIcon ? `13px ${ITEM_PADDING} 13px 13px` : `13px ${ITEM_PADDING}`,
         margin: 0,
-        border: 0,
-        background: backgroundColor,
+        border: `${borderWidthBase} solid ${borderColor}`,
+        outline: 0,
+        backgroundColor: 'transparent',
         color: buttonColor,
         ...textSmallStyle,
-        ...getFocusJssStyle({ color: primaryColor }),
+        overflowWrap: 'normal',
+        position: 'relative',
+        borderRadius: borderRadiusSmall,
+        '&::before': {
+          content: '""',
+          position: 'absolute',
+          ...getInsetJssStyle(-5),
+          border: `${borderWidthBase} solid transparent`,
+          borderRadius: '7px',
```

Comment:
why can't we use borderMedium

Reply:
<img width="289" alt="Bildschirm­foto 2023-02-07 um 09 36 51" src="https://user-images.githubusercontent.com/2981884/217193928-fa892846-cd2c-477a-a935-fceb070b0617.png">
<img width="241" alt="Bildschirm­foto 2023-02-07 um 09 37 14" src="https://user-images.githubusercontent.com/2981884/217193939-dfb78930-815a-45b9-b4f5-8884eff33647.png">

@stephanschroeter I personally think that it looks better with 7px border radius, rather than 8px, please see screenshots. If you think it's okay with 8px, then I will change it to borderRadiusMedium